xiblint/rules/color_assets.py

```diff
@@ -80,3 +102,9 @@ def _load_component(self, components, key):
             return float(string)
 
         return float(string) / 255.0
+
+    def _compare_colors(self, lhs, rhs):
+        # Compare with only 4 decimal places
+        return "{:.4f},{:.4f},{:.4f},{:.4f}".format(
```

Comment:
nit - `all(format(l, '.4f') == format(r, '.4f') for l, r in zip(lhs, rhs))`

Reply:
This doesn't work. Updating to use this says every color is invalid.